content/applications/general/email_communication/email_template.rst

```diff
@@ -1,98 +1,222 @@
+
+To use the :guilabel:`Dynamic Placeholder Generator` navigate to the :guilabel:`Dynamic Placeholder
+Generator` tab. Select the :guilabel:`Field` that the *dynamic placeholder* should reference by
+clicking on the :guilabel:`Field` drop-down. Should a :guilabel:`Sub-Model` or :guilabel:`Sub-Field`
+need to be specified, set those by selecting the corresponding drop-down. The *dynamic placeholder*
+will appear in the :guilabel:`Placeholder Expression` field. Copy and paste the *dynamic
+placeholder* in the appropriate field, as listed above.
+
+Default reply on email templates
+================================
+
+Under the :guilabel:`Email Configuration` tab on an email template, there is a :guilabel:`Reply To`
+field. In this field, email addresses can be added, to which replies are redirected when sending
+emails in mass using that template.
 
 .. image:: email_template/reply-to-template-sales.png
    :align: center
    :alt: Reply-to field on template.
 
-Because of this, setting a value in this field is useless as the value defined will be totally
-ignored. The default *reply-to* value is the default catchall email address to ensure a
-communication between your customer and your Odoo database. For more information about the way
-the catchall works, please check :ref:`how to manage inbound messages <email_communication/inbound_messages>`.
+The :guilabel:`Reply To` field is **only** used for mass mailing endeavors (sending emails in bulk).
+Bulk emails can be sent in almost every Odoo application that has a `List` view option.
+
+To send mass mails (while in `List` view), select the desired records where the emails are to be
+sent, click the :guilabel:`Action` button, represented by a `Gear` icon (:guilabel:` ⚙️`), and
+select the desired email option from the :guilabel:`Action` drop-down menu. If it is possible to
+send an email, a mail composer pop-up appears, with values that can be defined and customized.
+
+.. image:: email_template/composer-mass-mailing.png
+   :align: center
+   :alt: Composer in mass mailing mode after selecting multiple quotations.
 
-Transactional emails and corresponding URL for each company
-===========================================================
+Transactional emails and corresponding URLs
+===========================================
 
-When using Odoo, multiple events trigger the sending of automated emails. These emails are known
-as transactional emails and sometimes contain links pointing to your Odoo database.
+In Odoo, multiple events can trigger the sending of automated emails. These emails are known as
+*transactional emails*, and sometimes contain links redirecting to the Odoo database.
 
 By default, links generated by the database use the dynamic web.base.url key defined in the system
-parameters. More information about this :ref:`parameter <domain-name/web-base-url>`.
+parameters. For more information about this, see :ref:`parameter <domain-name/web-base-url>`.
```

Comment:
The "web.base.url" formatting here is an interesting one because technically it is a **System Parameter** _record_ with the **Key** field as `web.base.url`and the **Value** field as `https://custom-domain.com`.

(Sorry can't add a suggestion) I think it could be formatted in the following paragraphs first with italics and then no formatting after:

```
...the dynamic *web.base.url* key defined in the system parameters. For more...
```